docs/src/pages/[platform]/theming/css-only.mdx

```diff
@@ -0,0 +1,67 @@
+import { Fragment } from '@/components/Fragment';
```

Comment:
Should this be renamed to `authenticator-theming.mdx` since this really only shows how to theme the Authenticator at this time (given Angular and Vue don't have primitives yet)?